Pfm 72 dao recording

YefGav opened the pull request:
[Jira](https://mabelrincon.atlassian.net/browse/PFM-72)

Add RecordingRepository class to handle CRUD operations for Recording and update duration type.

Update duration type of RecordingTest to string an int

Grigory-Vladimiro reviewed (commented):
LGTM

MabelRincon reviewed (commented):
Te he dejado un par de comentarios, por si los quieres revisar y te parece bien cambiarlo. 
Salgo ahora, lo vuelvo a revisar cuando vuelva a la hora de la cena.

YefGav commented:
Thank you for your valuable feedback. You're a machine—really appreciate your keen eye for detail!

I have made the following changes based on your suggestions:

- Renamed the SQL column from date to recording_date and updated the corresponding attribute name in the Recording model from date to recordingDate to avoid conflicts with reserved keywords.

- Changed the type of the audioData attribute in the Recording model from String to byte[].

- Updated the DAO methods (now RecordingRepository) to use setBytes() and getBytes() for the audioData field.

MabelRincon reviewed (approved):
LGTM!
